app-doc/heirloom-doctools/heirloom-doctools-160308.ebuild

```diff
@@ -0,0 +1,90 @@
+EAPI=6
+
+inherit flag-o-matic toolchain-funcs multilib readme-gentoo
```

Comment:
You don't seem to be using multilib.eclass here. I suppose you had it for get_libdir but it's built-in since EAPi 6.

You want to use `readme.gentoo-r1`. Always use the newest version of an eclass. Also, you want to test your ebuilds since there is no such eclass as `readme-gentoo`.